EIPS/eip-3403.md

```diff
@@ -0,0 +1,138 @@
+| `0x60016000556001600055` | 20112 | 0| 0 | 1 |  1 |  |  20112 |
+| `0x60006000556000600055` | 3012 | 15000| 1 | 0 |  0 |  |  -11988 |
+| `0x60006000556001600055` | 3012 | 2800| 1 | 0 |  1 |  |  212 |
+| `0x60006000556002600055` | 3012 | 0| 1 | 0 |  2 |  |  3012 |
+| `0x60026000556000600055` | 3012 | 15000| 1 | 2 |  0 |  |  -11988 |
+| `0x60026000556003600055` | 3012 | 0| 1 | 2 |  3 |  |  3012 |
+| `0x60026000556001600055` | 3012 | 2800| 1 | 2 |  1 |  |  212 |
+| `0x60026000556002600055` | 3012 | 0| 1 | 2 |  2 |  |  3012 |
+| `0x60016000556000600055` | 3012 | 15000| 1 | 1 |  0 |  |  -11988 |
+| `0x60016000556002600055` | 3012 | 0| 1 | 1 |  2 |  |  3012 |
+| `0x60016000556001600055` | 212 | 0| 1 | 1 |  1 |  |  212 |
+| `0x600160005560006000556001600055` | 40118 | 19900| 0 | 1 |  0 |  1 |  20218 |
+| `0x600060005560016000556000600055` | 5918 | 17800| 1 | 0 |  1 |  0 |  -11882 |
+
+### With EIP-3403 partial refunds 
+
+If refunds were to be partially removed, as specified [here](https://github.com/ethereum/EIPs/pull/3403/), this would be the comparative table. **This table also assumes touched storage slots were already 'hot'**.
+
+| Code | Used Gas | Refund | Original | 1st | 2nd | 3rd | Effective gas (after refund)
+| -- | -- | -- | -- | -- | -- | -- | -- | 
+| `0x60006000556000600055` | 212 | 0| 0 | 0 |  0 |  |  212 |
+| `0x60006000556001600055` | 20112 | 0| 0 | 0 |  1 |  |  20112 |
+| `0x60016000556000600055` | 20112 | 19000| 0 | 1 |  0 |  |  1112 |
+| `0x60016000556002600055` | 20112 | 0| 0 | 1 |  2 |  |  20112 |
+| `0x60016000556001600055` | 20112 | 0| 0 | 1 |  1 |  |  20112 |
+| `0x60006000556000600055` | 3012 | 0| 1 | 0 |  0 |  |  3012 |
+| `0x60006000556001600055` | 3012 | 0| 1 | 0 |  1 |  |  3012 |
+| `0x60006000556002600055` | 3012 | 0| 1 | 0 |  2 |  |  3012 |
+| `0x60026000556000600055` | 3012 | 0| 1 | 2 |  0 |  |  3012 |
+| `0x60026000556003600055` | 3012 | 0| 1 | 2 |  3 |  |  3012 |
+| `0x60026000556001600055` | 3012 | 0| 1 | 2 |  1 |  |  3012 |
+| `0x60026000556002600055` | 3012 | 0| 1 | 2 |  2 |  |  3012 |
+| `0x60016000556000600055` | 3012 | 0| 1 | 1 |  0 |  |  3012 |
+| `0x60016000556002600055` | 3012 | 0| 1 | 1 |  2 |  |  3012 |
+| `0x60016000556001600055` | 212 | 0| 1 | 1 |  1 |  |  212 |
+| `0x600160005560006000556001600055` | 40118 | 19000| 0 | 1 |  0 |  1 |  21118 |
+| `0x600060005560016000556000600055` | 5918 | 0| 1 | 0 |  1 |  0 |  5918 |
+
+## The mutex usecase
+
+There are two typical ways to implement mutexes: '0-1-0' and '1-2-1. Let's see how they differ
+
+- '0-1-0':
+  - Istanbul: 1612
+  - Berlin: 212
+  - NoRefund: 20112
+  - EIP-3403: 1112
+- '1-2-1':
+  - Istanbul: 1612
+  - Berlin: 212
+  - NoRefund: 3012
+  - EIP-3403: 3012
+
+
+**Note**: In reality, there are never a negative gas cost, since the refund is capped at 0.5 * gasUsed. 
+However, these tables show the negative values, since a more real-world scenario would likely spend the 
+extra gas on other operations.'
+
```

Comment:
This content should be in the `## Motivation` section (the "why" section aka the "use cases" section).